### ISSUE TITLE
[tvheadend] Expose HTSP Port

### ISSUE THREAD

**Issue by koeberlue** [k8s-at-home/charts#467]
# Details

**Helm chart name:**

`tvheadend`

**Describe the solution you'd like:**

`tvheadend` usually exposes HTSP for IPTV on Port 9983. It would be great if the helm chart could expose this port as nodePort or hostPort as well.

**Additional Information:**

https://tvheadend.org/projects/tvheadend/wiki/Install_and_initial_setup (see Open Ports)

**Comment by onedr0p** [comment]
@koeberlue this should already be possible using `additionalServices` from the common library. 

@mr-onion-2  do you think this is a good idea to enable by default in the chart?

**Comment by mr-onion-2** [comment]
Ha I did actually add that as a default but then I thought perhaps it would be better set by the user if required and I removed it. I guess there's no harm having it as a default though? Happy to add back - your call.

**Comment by onedr0p** [comment]
If it's 100% needed for the app to function as expected you can add it, otherwise you can add it in the values.yaml but comment it out.

**Comment by onedr0p** [comment]
@mr-onion-2 until the PR is opened and merged, do you want to show post the YAML here for how you are doing it?

**Comment by mr-onion-2** [comment]
Hi, yeh all I'd done originally was add it as an additionalPort rather than an additionalService. If that's OK?

So this, but commented out 
```
service:
  port:
    port: 9981
  additionalPorts:
    port:
      port: 9982
      name: htsp
```

**Comment by bjw-s** [comment]
Yup, that seems fine! 😄

**Comment by bjw-s** [comment]
Implemented by #476
